assignment1/prime_number.rb

```diff
@@ -0,0 +1,15 @@
+def prime_numbers num_arr
+    num_arr.each {|num|
+        count = 0
+        2.upto(num) do |i|
```

Comment:
loop to number is not required since the optimal solution uses a loop till the integer part of the square root of the given value n. For eg., if n=16 then the loop should run from 2 to 4 only.